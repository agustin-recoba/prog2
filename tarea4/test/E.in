binarioBalanceado 10 10 11 12 13 14 15 16 17 18 19
imprimirBinario
apilar 7
apilar 6
apilar 5
cima
desapilar 
cima
estaVaciaPila
estaLlenaPila
recorridaPorNiveles
encolar 20
encolar 11
frente
desencolar
frente
Fin
