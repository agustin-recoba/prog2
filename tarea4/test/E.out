1>
2>
---(19,0.00)
--(18,0.00)
-(17,0.00)
---(16,0.00)
--(15,0.00)
(14,0.00)
---(13,0.00)
--(12,0.00)
-(11,0.00)
--(10,0.00)
3>Apilado.
4>Apilado.
5>Apilado.
6>5.
7>Desapilado.
8>6.
9>p NO está vacia.
10>p NO está llena.
11>13 16 19 | 10 12 15 18 | 11 17 | 14 
12>Encolado.
13>Encolado.
14>Binario vacío.
15>Desencolado.
16>11
17>Fin.
